builder/tart/step_run.go

```diff
@@ -133,140 +126,6 @@ func (s *stepRun) Cleanup(state multistep.StateBag) {
 	_, _ = cmd.Process.Wait()
 }
 
-func typeBootCommandOverVNC(
```

Comment:
Would it be possible to avoid moving this function and reduce the number changes made to this file?

This would greatly simplify the review process as it'd be clear what was actually changed.

Reply:
The first commit in the series (https://github.com/cirruslabs/packer-plugin-tart/pull/178/commits/f7dd2bfcd28111a3eb656cffecddf3fdd8fce9e4) just moves the code (sans some minor language changes to the log messages), so the commits on top of that change all represent the actual changes. 

I can remove the language changes if you want, so that the commit is a pure move of the function?

Reply:
Ie, the change (PR) is split into atomic commits, so it helps to review them individually, rather than the total change (https://github.com/cirruslabs/packer-plugin-tart/pull/178/files).

Reply:
>I can remove the language changes if you want, so that the commit is a pure move of the function?

Since we squash all changes anyways I think it'd be better if you could just avoid moving this function altogether.

And these "some minor language changes" that were made as a part of moving the function would be instantly visible.

Reply:
Sure, if that's how you prefer to review I can surely adopt that approach 😃

Reply:
To clarify, do you want me to still add the new logic to `vnc.go`, but keep `typeBootCommandOverVNC` in `step_run.go`, or to add all the new logic to `step_run.go` without any new file?

Reply:
> To clarify, do you want me to still add the new logic to `vnc.go`, but keep `typeBootCommandOverVNC` in `step_run.go`, or to add all the new logic to `step_run.go` without any new file?

That's up to you 🙌

Reply:
👍🏻

I'm investigating some details in how this behaves before pushing a new PR version. For some reason `bootcommand` sometimes feeds us keys with keycode `0` that doesn't match whatever the input text was. As this might result in looking for the wrong text I need to figure out what's going wrong.

As this is coming from `bootcommand`'s grammar, I don't suppose you've seen similar issues where a boot command button press is somehow "lost"?

Reply:
> As this is coming from `bootcommand`'s grammar, I don't suppose you've seen similar issues where a boot command button press is somehow "lost"?

No, I don't think I've encountered this while developing our [`boot_command`](https://github.com/cirruslabs/macos-image-templates/blob/c18ef9c5530d80bd2d728c4ffb617e9b8993009e/templates/vanilla-sequoia.pkr.hcl#L19).